documentation/book/ref-storage-persistent.adoc

```diff
@@ -0,0 +1,76 @@
+// Module included in the following assemblies:
+//
+// assembly-storage.adoc
+
+[id='ref-persistent-storage-{context}']
+= Persistent storage
+
+Persistent storage uses {K8sPersistentVolumeClaims} to provision persistent volumes for storing data.
+Persistent Volume Claims can be used to provision volumes of many different types, depending on the {K8SStorageClass} which will provision the volume.
+The data types which can be used with persistent volume claims include many types of SAN storage as well as {K8sLocalPersistentVolumes}.
+
+To use persistent storage, the `type` has to be set to `persistent-claim`.
+Persistent storage supports additional configuration options:
+
+`size` (required)::
+Defines the size of the persistent volume claim, for example, "1000Gi".
+
+`class` (optional)::
+The {ProductPlatformName} {K8SStorageClass} to use for dynamic volume provisioning.
+
+`selector` (optional)::
+Allows selecting a specific persistent volume to use.
+It contains a `matchLabels` field which contains key:value pairs representing labels for selecting such a volume.
+
+`delete-claim` (optional)::
+Boolean value which specifies if the Persistent Volume Claim has to be deleted when the cluster is undeployed.
+Default is `false`.
+
+.Example fragment of persistent storage configuration with 1000Gi `size`
+[source,yaml]
+----
+# ...
+storage:
+  type: persistent-claim
+  size: 1000Gi
+# ...
+----
+
+The following example demonstrates the use of a storage class.
+
+.Example fragment of persistent storage configuration with specific Storage Class
+[source,yaml,subs="attributes+"]
+----
+# ...
+storage:
+  type: persistent-claim
+  size: 1Gi
+  class: my-storage-class
+# ...
+----
+
+Finally, a `selector` can be used to select a specific labeled persistent volume to provide needed features such as an SSD.
```

Comment:
labeled > labelled

Reply:
So, both are correct, but the single L is the 'American' way and since we use mostly American spelling I changed this. 

https://www.grammarly.com/blog/labeled-labelled/